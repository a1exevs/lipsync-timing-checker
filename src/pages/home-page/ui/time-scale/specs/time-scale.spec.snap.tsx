import { render } from '@testing-library/react';
import React from 'react';

import TimeScale from 'src/pages/home-page/ui/time-scale/time-scale';

describe('TimeScale', () => {
  const defaultProps = {
    durationSec: 10,
    timelineWidthPx: 1000,
    scrollLeftPx: 0,
    viewportWidthPx: 500,
    heightPx: 50,
  };

  it('matches snapshot (default)', () => {
    const { container } = render(<TimeScale {...defaultProps} />);
    expect(container).toMatchSnapshot();
  });

  it('matches snapshot (scrolled)', () => {
    const { container } = render(<TimeScale {...defaultProps} scrollLeftPx={100} />);
    expect(container).toMatchSnapshot();
  });
});
