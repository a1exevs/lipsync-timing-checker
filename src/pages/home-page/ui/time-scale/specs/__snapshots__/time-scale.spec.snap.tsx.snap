// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`TimeScale matches snapshot (default) 1`] = `
<div>
  <section
    class="flex overflow-x-auto overflow-y-hidden border border-gray-300"
    style="scrollbar-width: none;"
  >
    <canvas
      height="50"
      style="display: block; width: 500px; height: 50px;"
      width="500"
    />
  </section>
</div>
`;

exports[`TimeScale matches snapshot (scrolled) 1`] = `
<div>
  <section
    class="flex overflow-x-auto overflow-y-hidden border border-gray-300"
    style="scrollbar-width: none;"
  >
    <canvas
      height="50"
      style="display: block; width: 500px; height: 50px;"
      width="500"
    />
  </section>
</div>
`;
